### ISSUE TITLE
Add ability to annotate entire class with since('version')

### ISSUE THREAD

**Issue by ptnapoleon** [riptano/cassandra-dtest#152]

**Comment by mambocab** [comment]
Unless I misunderstand the problem, this already works. Demo'd on a branch over here:

https://github.com/mambocab/cassandra-dtest/blob/skip-class/test_skip_class.py

```
nosetests -v test_skip_class.py

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```

`@since` calls its argument's `skip` method, which is added to the `Tester` class [here](https://github.com/riptano/cassandra-dtest/blob/master/dtest.py#L451).

**Comment by ptnapoleon** [comment]
Does it compare version properly? I swear I tried it and it didn't work but if it does, cool!

**Comment by ptnapoleon** [comment]
It does not work for me, and I suspect you have a test discoverability issue. If the test1 had been run, but skipped, it would say `Ran 1 test in 0.000s OK (SKIP=1)`. So for some reason it cannot find your test. The error message I am getting is 

Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Library/Python/2.7/site-packages/nose/util.py", line 619, in newfunc
    return func(*arg, **kw)
TypeError: wrapped() takes exactly 1 argument (0 given)

**Comment by mambocab** [comment]
My bad -- thanks for debugging. I've got a more robust solution; putting together a PR now.

**Comment by mambocab** [comment]
Closed with #176.
